File: sept15/lowercase.js
arr = ["Totam", "ut", "odit", "quis", "Maiores", "unde", "EX", "EST", "corporis"]

arr.forEach( function(eachString) {
  if (eachString.length > 4 && eachString == eachString.toLowerCase()) {
    console.log("long and lowercase");
  } else if (eachString.length > 4) {
    console.log("long");
  } else if (eachString == eachString.toLowerCase()) {
    console.log("lowercase");
  } else {
    console.log(eachString);
  }
})

File: sept15/lowercase.rb
arr = ["Totam", "ut", "odit", "quis", "Maiores", "unde", "EX", "EST", "corporis"]

arr.each do | each_string |
  if each_string.length > 4 && each_string == each_string.downcase
    puts "long and lowercase"
  elsif each_string.length > 4
    puts "long"
  elsif each_string == each_string.downcase
    puts "lowercase"
  else
    puts each_string
  end
end
